CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## 2.0.0 (Unreleased)
+* Use [AWS-LC](https://github.com/awslabs/aws-lc/) as the backing native cryptography library for ACCP
+* Track the AWS-LC dependency as a git submodule instead of downloaded tarball
+* Drop support for (non-EC) DSA signatures
+* Use AWS-LC's DRBG implementation, drop custom java implementation
+* Drop support for `secp192r1`, as well as other non-NIST "legacy" curves
+
 ## 1.7.0 (Unreleased)
```

Comment:
Do we think that 1.7.0 will ever actually exist? If it won't, merge it into the 2.0.0 notes.

Reply:
i don't se  a particular reason why we'd do a 1.7.0 release at this point. I'll pull up the interesting and still-relevant bits into 2.0.0's section.